libdirections-offboard/src/test/java/com/mapbox/navigation/route/offboard/router/NavigationWalkingOptionsTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.navigation.route.offboard.router
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class NavigationWalkingOptionsTest {
```

Comment:
I see the walking options are still here.

Reply:
yep, had a chat with @Guardiola31337, walked option should exist